packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -235,10 +238,12 @@ export class RunningSummarizer implements IDisposable {
 			this.mc.logger,
 		);
 
-		// Listen for ops
-		this.runtime.deltaManager.on("op", (op) => {
-			this.handleOp(op);
-		});
+		if (listenToDeltaManagerOps) {
```

Comment:
can you please add some comments on when listenToDeltaManagerOps is true, and overall purpose of this argument and expectations of caller and overall flow?